src/scripts/__tests__/uniquePairing.test.js

```diff
@@ -0,0 +1,63 @@
+import shuffle from "../../util/shuffle.js";
+import { isValidPair, uniquePairingHelper } from "../filters.js";
+import pair from "../../util/pair.js";
+import { readFileSync } from "fs";
+import { fileURLToPath } from "url";
+import { dirname, join } from "path";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+const internsPath = join(__dirname, "../../documents/interns.json");
+const internInfo = JSON.parse(readFileSync(internsPath, "utf-8"));
+
+async function pairInterns(isUniqueDept, isUniqueLoc) {
+  let interns = [];
```

Comment:
Don't think interns gets reassigned so this can be a const
```suggestion
  const interns = [];
```